apps/pie-docs/src/assets/styles/components/_card.scss

```
@@ -0,0 +1,67 @@
+@use '../fozzie-config' as f;
+
+$card-width: 328px;
```

do you think we should rename this to: 
```suggestion
$card-max-width: 328px;
```